tasks/vm_state_present.yml

```diff
@@ -17,7 +17,7 @@
 - name: Manage disks
   ovirt_disk:
     auth: "{{ ovirt_auth }}"
-    name: "{{ item.0.name }}_{{ item.1.name }}"
+    name: "{{ item.1.name }}"
```

Comment:
We shouldn't solve it like this, because user which will execute old playbook with newer role version will get new disks created. This breaks functionality.

Reply:
What if I would add new variable like  `template_disk: true`?

Reply:
I think we just need check if disk with specified name exists, if yes then update it as user request, if not - create it as user request with specified parameters and prefix.

Reply:
as commented on #24, I would prefer a new variable like "namePrefix" with default "true" to secure exiting functionality but allow more control over the actual disk name.

Reply:
Correct, that would be the best solution. @mencas can you please do it?

Reply:
I tried:
    name: "{% if item.1.namePrefix | default(true) %}{{ item.0.name }}_{% endif %}{{ item.1.name }}"
works like a charm!